app/api/v1/revalidate/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { revalidateTag } from 'next/cache';
+
+export async function GET(request: NextRequest) {
```

Comment:
This route can be utilised by our backend to invalidate on publish event for data changes. Ideally this would be guarded by auth.